src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php

```diff
@@ -104,7 +104,8 @@ private static function getPhpUnitCliConfigArgument()
                 $dir = realpath($reversedArgs[$argIndex - 1]);
                 break;
             } elseif (strpos($testArg, '--configuration=') === 0) {
-                $argPath = substr($testArg, strlen('--configuration='));
+                // strlen('--configuration=') = 16
+                $argPath = substr($testArg, 16);
```

Comment:
This is not a Symfony test. This is the base class for users' tests. Note the namespace Test rather than Tests